docker/dev/php-fpm/Dockerfile

```diff
@@ -1,23 +1,15 @@
-FROM phpdockerio/php74-fpm:latest
-WORKDIR "/application"
+FROM bref/php-74-fpm-dev:latest
 
-# Fix debconf warnings upon build
-ARG DEBIAN_FRONTEND=noninteractive
+USER root
 
-# Install selected extensions and other stuff
-RUN apt-get update \
-    && apt-get -y --no-install-recommends install php-redis php-xdebug php7.4-bcmath php7.4-gd php7.4-intl php-msgpack php-yaml \
-    && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
-
-# Install git
-RUN apt-get update \
-    && apt-get -y install git \
-    && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
-    
 RUN mkdir -p /home/dev/.composer \
     && chown -R 1000:1000 /home/dev/
 
+RUN mkdir -p /tmp/cache \
+    && chown -R 1000:1000 /tmp/cache
+
 ENV COMPOSER_CACHE_DIR='/home/dev/.composer'
 
 # Install prestissimo
```

Comment:
`# Install composer and prestissimo`?

Reply:
is prestissimoreally needed? can't we rely on symfony flex?

Reply:
prestissimo speeds up the installation of dependencies, it has nothing to do with the application software!

Reply:
Yep I know prestissimo, we used it  for years .. in Musement we removed it as soon as we included symfony flex anyway

Reply:
sorry but I don't understand the combination `prestissimo` => `symfony/flex`
How `symfony/flex` can replace `prestissimo`?
Can you explain it to me?
Thanks

Reply:
@JellyBellyDev Flex is already performing parallel downloading

Reply:
Tell him @garak, tell him ;)

Reply:
@garak link please, otherwise I don't believe it! :P

Reply:
https://github.com/symfony/flex/blob/master/src/ParallelDownloader.php

Reply:
but does it come into play only for packages that have recipes or for all dependencies?

Reply:
just try it yourself: do a `composer update` on one of your projects with Flex and see that everything is parallelized

Reply:
ok as soon as I can try it!
So are you proposing to remove `prestissimo` because it's useless?

Reply:
it's nnot useless in fact, it helps creating long threads on GH :D